contracts/Quest.sol

```diff
@@ -100,17 +116,25 @@ contract Quest is Ownable, IQuest {
         emit Claimed(msg.sender, totalRedeemableRewards);
```

Comment:
These lines are marked as unreachable (cause `_calculateRewards()` reverts here lol)

Don't think it'll be an issue on child contracts, but want to make sure

<img width="611" alt="CleanShot 2023-01-22 at 02 33 06@2x" src="https://user-images.githubusercontent.com/8920301/213911478-844bdf79-67c8-441b-8bb0-be839e2db1c5.png">

Reply:
Yeah we should consider adding those functions to the interface instead tbh. I just had reverts there bc that's a common web2 pattern to force implementation